x-pack/plugins/observability_solution/inventory/public/components/routing/root.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RedirectAppLinks } from '@kbn/shared-ux-link-redirect-app';
+import React from 'react';
+import { type AppMountParameters, type CoreStart } from '@kbn/core/public';
+import { RouteRenderer, RouterProvider } from '@kbn/typed-react-router-config';
+import { HeaderMenuPortal } from '@kbn/observability-shared-plugin/public';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { InventoryContextProvider } from '../inventory_context_provider';
+import { inventoryRouter } from '../../routes/config';
+import { HeaderActionMenuItems } from './header_action_menu';
+import { InventoryStartDependencies } from '../../types';
+import { InventoryServices } from '../../services/types';
+
+export function AppRoot({
```

Comment:
Just to understand if there is no confusion. There is another `router` folder containing the route configuration. Maybe this `routing` folder can be renamed to avoid confusion

Reply:
ahhhh.. yes, we definitely need to improve this folder name.

Reply:
will do!